sdk/digitaltwins/azure-digitaltwins-core/README.md

```diff
@@ -0,0 +1,399 @@
+# Azure Azure Digital Twins Core client library for Python
+
+This package contains an SDK for Azure Digital Twins API to provide access to the Azure Digital Twins service for managing twins, models, relationships, etc.
+
+This package contains an isomorphic SDK for Azure Digital Twins API to provide access to the Azure Digital Twins service for managing twins, models, relationships, etc.
+
+## Getting started
```

Comment:
Awesome! Thanks for adding all the snippets to the readme - it really helps customers get started :)

Reply:
You are welcome!  :)